com.adsbynimbus.nimbus/Editor/IOSBuildDependencies.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace Nimbus.Editor {
 	public class IOSBuildDependencies {
-		private const string SdkVersion = "2.1.0";
+		private const string SdkVersion = "2.8.1";
```

Comment:
Curious any reason why 2.8.1 and not 2.8.0 like Android or why isn't Android on 2.8.1??

Reply:
iOS had a patch release so I had updated to the latest for each minor verison; this has been subsequently updated to 2.10.0 on both platforms.